scripts/build-binary.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -e
+
+YORKIE_VERSION=$1
+
+if [ -z "$1" ]; then
+  echo "Usage: ${0} VERSION" >> /dev/stderr
+  exit 255
+fi
+
+set -u
+
+function main {
+  mkdir -p release
+  cd release
+
+  tarcmd=tar
```

Comment:
Is there any reason to manage it as a separate variable?

Reply:
No, there is no particular reason.
Should I remove the variable?